src/cli.ts

```diff
@@ -1,164 +1,82 @@
-  if (input === 'preconnect' || input === 'jitp') {
-    return input;
-  }
-
-  new Log(false).error(
-    'Input for onboard must be blank, "jitp" or "preconnect"',
-  );
-  process.exit();
-};
-
-const getConfig = (env: any, args: string[]): SimulatorConfig =>
+const sensorOptions = ['gps', 'gnss', 'acc', 'temp', 'device', 'rsrp', 'location', 'log', 'alert'];
+const getConfig = (env: any, args: string[]) =>
   program
     .requiredOption(
       '-k, --api-key <apiKey>',
-      'API key for nRF Cloud',
+      'nRF Cloud REST API Key. May be set with the API_KEY environment variable.',
       env.API_KEY,
     )
     .option(
-      '-c, --certs-response <certsResponse>',
-      'JSON returned by call to the Device API endpoint: POST /devices/{deviceid}/certificates',
-      env.CERTS_RESPONSE,
+      '-h, --api-host <apiHost>',
+      'nRF Cloud API Host. May be set with the API_HOST environment variable.',
+      env.API_HOST ? env.API_HOST : 'https://api.nrfcloud.com',
     )
     .option(
-      '-e, --endpoint <endpoint>',
-      'AWS IoT MQTT endpoint',
-      env.MQTT_ENDPOINT,
+      '-d, --device-id <deviceId>',
+      'ID of the device. May be set with the DEVICE_ID environment variable. If you pass a device ID, an attempt is made to find locally stored device credentials in an adjacent "credentials" folder that is auto-created for you. Otherwise, a new device is created and added to your team, with device credentials stored locally for easy reuse.',
+      env.DEVICE_ID,
     )
-    .option('-d, --device-id <deviceId>', 'ID of the device', env.DEVICE_ID)
-    .option(
-      '-o, --device-ownership-code <deviceOwnershipCode>',
-      'PIN/ownership code of the device',
-      env.DEVICE_OWNERSHIP_CODE || '123456',
+    .addOption(
+      new Option(
+        '-t, --device-type <deviceType>',
+        'Specifies the type of device you want created. In most cases you will use the default "Generic" device. An "MQTT Team" device is mainly for debugging, or for use in an MQTT bridge. See https://docs.nordicsemi.com for more information.',
+      ).choices(['Generic', 'Team']).default('Generic'),
     )
-    .option(
-      '-m, --mqtt-messages-prefix <mqttMessagesPrefix>',
-      'The prefix for the MQTT unique to this tenant for sending and receiving device messages',
-      env.MQTT_MESSAGES_PREFIX,
+    .addOption(
+      new Option(
+        '-c, --certificate-type <certificateType>',
+        'Specifies the type of certificate you want created for your new device: self-signed certificate or one for Just-In-Time-Provisioning (JITP). The latter is discouraged, and is mainly for internal Nordic Semiconductor use.',
+      ).choices(['self-signed', 'JITP']).default('self-signed'),
     )
     .option(
-      '-s, --services <services>',
-      'Comma-delimited list of services to enable. Any of: [gps,gnss,acc,temp,device,rsrp,location,log,alert]',
+      '-p, --prevent-association <preventAssociation>',
```

Comment:
Should this just take a `-p` to be true? `-p true` feels wrong.
```suggestion
      '-p, --prevent-association',
```

Reply:
>Maybe if a user already generated device certs they could add them to the /credentials before running the simulator

Yes, that's the idea. Allows total flexibility, as well as easier visibility about what devices you have the creds for.